sys/freebsd/sys.txt

```diff
@@ -197,6 +210,20 @@ rtprio {
 	prio	int16[0:RTP_PRIO_MAX]
 }
 
+thr_param {
+	start_func	ptr[in, text[target]]
+	arg		intptr
+	stack_base	ptr64[in, array[int8]]
+	stack_size	bytesize[stack_base, intptr]
+	tls_base	ptr64[in, array[int8]]
+	tls_size	bytesize[tls_base, intptr]
+	child_tid	ptr[out, lwpid]
+	lwpid		ptr[out, lwpid]
+	flags		flags[thr_flags, int32]
```

Comment:
With a 64-bit data model there will be a four byte hole following the flags field. I'm not sure how we can express that. Note that we fuzz the 32-bit compat layer as well.

Reply:
Is it just a normal padding in a C struct?
If so syzkaller natively supports that, it lays out struct as C compiler would do.

Reply:
Yes, it's just regular padding. I didn't realize that syzkaller deals with that without any hand-holding, so I guess there's no problem.